### PR TITLE
fixed color picker positioning

### PR DESCRIPTION
**Pull request by melowsoft**
- [ ] **Update release notes if necessary**

- fixed the colour picker not appearing in the right place

### CODE REVIEW

File: client/src/components/common/ColorPicker.jsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ import LegendShape from '../charts/LegendShape';
 const ColorPicker = ({ title, left = 0, top = 0, placement, style, hideArrow, ...rest }) => (
   <Popover
     left={left}
-    top={top}
+    top={0}
```
Comment (kardan) [inline client/src/components/common/ColorPicker.jsx:12]: This component is used for several types of charts (see https://github.com/akvo/akvo-lumen/search?q=ColorPicker&type=code) I would guess this change will be a bit off for some chart types, or was that taken into consideration and it all works fine?

Comment (melowsoft) [reply client/src/components/common/ColorPicker.jsx:12]: Okay, I would effect it, just for the simple bar chart and see

Comment (melowsoft) [reply client/src/components/common/ColorPicker.jsx]: I have made the change just for the Simple bar chart component

Comment (kardan) [reply client/src/components/common/ColorPicker.jsx]: Cool!

**Pull request merged**